packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx

```diff
@@ -153,14 +156,14 @@ export function PanelChrome({
         <div className={styles.rightAligned}>
           {menu && (
             <Dropdown overlay={menu} placement="bottom">
-              <div className={cx(styles.item, styles.menuItem, 'menu-icon')} data-testid="menu-icon" style={itemStyles}>
-                <IconButton
-                  ariaLabel={`Menu for panel with ${title ? `title ${title}` : 'no title'}`}
-                  tooltip="Menu"
-                  name="ellipsis-v"
-                  size="sm"
-                />
-              </div>
+              <Button
```

Comment:
We need to add the squared focus style instead of the rounded one, maybe we can add this inside menuItem class? 

![image](https://user-images.githubusercontent.com/239999/212104252-6da27aeb-0d04-4f6f-a87f-8715917cadf1.png)

Reply:
@axelavargas not sure why? what is wrong with the default focus style? 
<img width="379" alt="Screenshot 2023-01-12 at 15 23 35" src="https://user-images.githubusercontent.com/10999/212107713-d84d115c-e7bc-4d0e-841a-f1c17183b46d.png">

Reply:
Nothing is wrong with this 😅 , but the focus style suggested in the [design](https://www.figma.com/file/jdTCXlq75p55itRRSaerFx/Panel-header-and-menu?node-id=2672%3A235&t=MpAO4eFmEHl9b5P4-0) has no rounded borders. 

And in the title items PR, we followed that suggestion, so it will look inconsistent if we have this with rounded borders.

Reply:
@axelavargas lol, probably due to figma and app is out of sync, I don't think @Ijin08 want's to change and have a unique special focus style for this that is different than all other focus styles 🤣

Reply:
> so it will look inconsistent if we have this with rounded borders.

@axelavargas you mean it will be inconsistent with every other focus style in the whole app if it's not rounded?

Reply:
🤦‍♀️I think you are right, I got so carried away following the design that I missed the fact that squared borders will look inconsistent in the whole app 😅.